playbooks/nginxplus.yml

```diff
@@ -21,10 +21,12 @@
 
   # updates existing load balancer
   roles:
-    - role: ../roles/nginxplus
+    - role: ../roles/nginxplusprotect
+    - role: ../roles/datadog
```

Comment:
Is there a reason this is now needed for staging?  Should it only be run in the production env?

Reply:
we only install datadog on staging. So if the playbook is run datadog will not be installed.